pkg/controller/queuejob_controller.go

```diff
@@ -279,8 +285,19 @@ func (qjm *QueueJobController) syncQueueJob(key string) error {
 	}
 
 	if job.Spec.AggrResources.Items != nil {
-		for i, ar := range job.Spec.AggrResources.Items {
-			qjm.qjobResControls[ar.Type].Sync(i, sharedJob, &ar)
+		for i := range job.Spec.AggrResources.Items {
+			qjm.refManager.AddTag(&job.Spec.AggrResources.Items[i], func() string {
```

Comment:
suggest adding error handling for `AddTag` when the return value is not nil

Reply:
ok